src/Maker/MakeAuthenticator.php

```diff
@@ -161,13 +157,6 @@ function ($answer) {
 
         $command->addOption('entry-point', null, InputOption::VALUE_OPTIONAL);
 
-        if (!$this->useSecurity52) {
-            $input->setOption(
-                'entry-point',
-                $interactiveSecurityHelper->guessEntryPoint($io, $securityData, $input->getArgument('authenticator-class'), $firewallName)
```

Comment:
Is `guessEntryPoint()` used at all anymore?

Reply:
good catch!